lib/consoletest.pm

```diff
@@ -16,5 +16,11 @@ sub post_run_hook {
     $self->clear_and_verify_console;
 }
 
+sub post_fail_hook {
+    # Export logs after failure
+    script_run "journalctl --no-pager -b 0 > /tmp/full_journal.log";
```

Comment:
"better not override but override?" I think my Oli parser is broken

@okurz please can you explain here the purpose of `$self->SUPER::post_fail_hook;`, how it should be used, and how using it would be better than what @kravciak proposes here?

Thanks